apps/cowswap-frontend/src/modules/ordersTable/pure/OrdersTableContainer/OrdersTabs.tsx

```diff
@@ -141,9 +145,10 @@ const TabButton = styled(Link)<{ active: string; isUnfillable?: boolean; isSigni
 
 export interface OrdersTabsProps {
   tabs: OrderTab[]
+  isWalletConnected?: boolean
```

Comment:
Why it can be undefined? Looks like it's always boolean

Reply:
Addressed this.